CONTRIBUTING.md

```diff
@@ -71,110 +44,57 @@ You can find the code complete and release dates of the next three Azure PowerSh
-
 ### Pull Request Guidelines
 
 A pull request template will automatically be included as a part of your PR. Please fill out the checklist as specified. Pull requests **will not be reviewed** unless they include a properly completed checklist.
 
-The following is a list of guidelines that pull requests opened in the Azure PowerShell repository must adhere to. You can find a more complete discussion of Azure PowerShell design guidelines [here](documentation/development-docs/azure-powershell-design-guidelines.md).
-
-#### Cleaning up Commits
-
-If you are thinking about making a large change to your Azure PowerShell cmdlets, **break up the change into small, logical, testable chunks, and organize your pull requests accordingly**.
-
-Often when a pull request is created with a large number of files changed and/or a large number of lines of code added and/or removed, GitHub will have a difficult time opening up the changes on their site. This forces the Azure PowerShell team to uses separate software, such as CodeFlow or Beyond Compare, to do a code review on the pull request.
+The following is a list of guidelines that pull requests opened in the Azure PowerShell repository must adhere to. You can find a more complete discussion of Azure PowerShell design guidelines [here](documentation/development-docs/design-guidelines).
 
-If you find yourself creating a pull request and are unable to see all the changes on GitHub, we recommend **splitting the pull request into multiple pull requests that are able to be reviewed on GitHub**.
+#### General guidelines
 
-If splitting up the pull request is not an option, we recommend **creating individual commits for different parts of the pull request, which can be reviewed individually on GitHub**.
+The following guidelines must be followed in **EVERY** pull request that is opened.
 
-For more information on cleaning up the commits in a pull request, such as how to rebase, squash, and cherry-pick, click [here](documentation/development-docs/cleaning-up-commits.md).
+- Title of the pull request is clear and informative
+- The appropriate `ChangeLog.md` file has been updated with a snippet describing the changes being made
+- There are a [small number of commits](documentation/development-docs/cleaning-up-commits.md) that each have an informative message
+- All files shipped with a module should contain a proper Microsoft license header
+- For public API changes to cmdlets:
+    - a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
+    - the markdown help files have been regenerated using the commands listed [here](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
 
```

Comment:
Should we link the PR reviewer check list to show a customer what to expect from a PR review?

Reply:
Ideally, everything that's in our reviewer checklist would be mentioned in this `Pull Request Guidelines` section, or found in one of the documents linked out in this section. If there's something in the reviewer checklist that's missing from this section, let me know and I'll update the correct document.